app/javascript/widget/components/UnreadMessage.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div class="chat-bubble-wrap">
-    <div class="chat-bubble agent">
+    <div
+      :class="['chat-bubble agent', checkMessageIsCampaign && 'campaign']"
```

Comment:
Can we follow the `{ 'x': true }` inside classes? Also the `chat-bubble agent` can be moved outside as a separate `class="chat-bubble agent"`